lib/pluginmanager/install.rb

```diff
@@ -8,6 +8,7 @@ class LogStash::PluginManager::Install < LogStash::PluginManager::Command
   option "--version", "VERSION", "version of the plugin to install"
   option "--[no-]verify", :flag, "verify plugin validity before installation", :default => true
   option "--development", :flag, "install all development dependencies of currently installed plugins", :default => false
+  option "--local", :flag, "install a plugin from a local cache", :default => false
```

Comment:
I don't think we should mention "cache" to users. can't we just say "install a local plugin" or something like that?